cardano-node/app/cardano-node.hs

```diff
@@ -123,24 +117,22 @@ initializeAllFeatures (NodeCLI topInfo nodeAddr protocol vMode logCLI traceOpts
 -- Parsers & Types
 -------------------------------------------------------------------------------
 
--- TODO: Condense `NodeCLI` into one big `PartialCardanoConfiguration`
-data NodeCLI = NodeCLI
-                !TopologyInfo
-                !NodeAddress
-                !Protocol
-                !ViewMode
-                !LoggingCLIArguments
-                !TraceOptions
-                !PartialCardanoConfiguration
+data NodeCLI = NodeCLI !PartialCardanoConfiguration
```

Comment:
Now, do we need this ADT anymore? : -)

Reply:
@deepfire If we decide that `cardano-cli` and `cardano-node` should have separate configurations then no. I'm leaning towards this and I was planning to remove it in a follow up PR.